test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs

```diff
@@ -99,9 +99,6 @@ public override void ConfigureJobHost(IWebJobsBuilder webJobsBuilder)
                     "CosmosDBIn",
                     "CosmosDBOut"
                 };
-
```

Comment:
removed all of the hardcoded initialization that was peppered throughout the code.